activestate.yaml

```diff
@@ -39,7 +39,9 @@ scripts:
     value: |
       set -e
       $constants.SET_ENV
-      go build -tags "$GO_BUILD_TAGS" -o $constants.BUILD_TARGET $constants.CLI_BUILDFLAGS $constants.CLI_PKGS
+      export GOARCH=${1:-amd64}
+      export TARGET_SUFFIX=${1:-}
+      go build -tags "$GO_BUILD_TAGS" -o $constants.BUILD_TARGET${TARGET_SUFFIX} $constants.CLI_BUILDFLAGS $constants.CLI_PKGS
```

Comment:
I think this would produce a build target `state.exe386`, no?

Reply:
Yes, I agree that looks silly.  The update-generator scripts renames it to `windows-386.exe`, but I will change it, and make it an explicit `build32` script with a separate `BUILD_TARGET`...  Just a little verbose...